src/app.ts

```diff
@@ -1,33 +1,47 @@
 import express from 'express'
-import { JwtAuthorizationMiddleware } from './types'
+import Session from 'express-session'
 import { quoteRouter } from './routes/quote'
 import { kycRouter } from './routes/kyc'
 import { accountsRouter } from './routes/accounts'
 import { transferRouter } from './routes/transfer'
 import { errorToStatusCode } from './middleware/error'
+import { authRouter } from './routes/auth'
+
+// this is typically the name of the provider
+const SESSION_NAME = 'api-starter'
+
+// a secret for signing the session token, typically loaded via env vars and not
+// stored in the codebase. See
+// https://www.npmjs.com/package/express-session-expire-timeout#secret for more
+// details.
+const SESSION_SECRET = process.env.SESSION_SECRET || 'api-starter-secret'
 
 export function initApp({
-  jwtAuthMiddleware,
   clientAuthMiddleware,
 }: {
-  jwtAuthMiddleware: JwtAuthorizationMiddleware
   clientAuthMiddleware: express.RequestHandler[]
 }): express.Application {
   const app = express()
 
   app.use(express.json())
-
-  app.use('/quote', quoteRouter({ jwtAuthMiddleware, clientAuthMiddleware }))
-  app.use('/kyc', kycRouter({ jwtAuthMiddleware, clientAuthMiddleware }))
   app.use(
-    '/accounts',
-    accountsRouter({ jwtAuthMiddleware, clientAuthMiddleware }),
-  )
-  app.use(
-    '/transfer',
-    transferRouter({ jwtAuthMiddleware, clientAuthMiddleware }),
+    // https://www.npmjs.com/package/express-session-expire-timeout#sessionoptions
+    Session({
+      name: SESSION_NAME,
+      secret: SESSION_SECRET,
+      resave: true,
+      saveUninitialized: true,
+      cookie: { secure: false, sameSite: true },
```

Comment:
I'm reading in the docs you linked to that `secure: true` is recommended in production